service/src/main/java/uk/gov/hmcts/reform/fpl/model/notify/c2uploaded/C2UploadedTemplate.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.reform.fpl.model.notify.c2uploaded;
+
+import lombok.Getter;
+import lombok.Setter;
+import uk.gov.hmcts.reform.fpl.model.notify.SharedNotifyTemplate;
+
+@Getter
+@Setter
+public final class C2UploadedTemplate extends SharedNotifyTemplate {
+    private String callout;
```

Comment:
Hmm, what do you think of callout? I know we use it in a few places but all it is just respondentLastName (which we have) and caseReference. Could we expand out the model's properties and build callout in the template itself? Or we could have a `buildCallout` in the `SharedNotifyTemplate`.

Thoughts?

Reply:
leaving for now as discussed